clang/test/CAS/depscan-dependency-file.c

```diff
@@ -0,0 +1,29 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: split-file %s %t
+
+// Baseline for comparison.
```

Comment:
Since the dependency is very simple for the test cases here, I prefer do direct file check on the `.d` file, instead of doing `diff` between two outputs. It is much easier to read the tests if I can directly read the expected output.